R/api.R

```diff
@@ -0,0 +1,201 @@
+#' Meetup API Prefix
+#'
+#' @keywords internal
+#' @noRd
+meetup_api_prefix <- function() {
+  Sys.getenv(
+    "MEETUP_API_URL",
+    "https://api.meetup.com/gql-ext"
+  )
+}
+
+#' Create and Configure a Meetup API Request
+#'
+#' This function prepares and configures an HTTP request for interacting with
+#' the Meetup API. It allows the user to authenticate via OAuth, specify the
+#' use of caching, and set custom client configuration.
+#'
+#' @param cache A logical value indicating whether to cache the OAuth token
+#'   on disk. Defaults to `TRUE`.
+#' @param ... Additional arguments passed to `meetup_client()` for setting up
```

Comment:
```suggestion
#' @param ... Additional arguments passed to [meetup_client()] for setting up
```

https://roxygen2.r-lib.org/articles/rd-formatting.html?q=link#default-link-text